### PR TITLE
Task/0002 cw do you speak english

### PR DESCRIPTION
**Pull request by skripchenkoVeta**

### CODE REVIEW

**Comment by mTeeeur** [comment]
Why do you want to merge into master? @skripchenkoVeta

**Review by mTeeeur — CHANGES_REQUESTED** [review]
Please fix linter issues

File: test/0002-CW-do-you-speak-english/Do-you-speak-English.mocha.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const assert = require('assert');
+const en = require('../../tasks/0002-CW-do-you-speak-english');
+
+describe('simple tests',()=>{
+    it('should return true',()=>{assert.strictEqual(en("englishassdsa"), true)});
+    it('should return true',()=>{assert.strictEqual(en("LiaEnGLiShAA"),true)});
```
Comment (Rascass) [inline test/0002-CW-do-you-speak-english/Do-you-speak-English.mocha.js:6]: why don't you use assert.ok() here?

**Review by Rascass — CHANGES_REQUESTED** [review]

File: test/0002-CW-do-you-speak-english/Do-you-speak-English.mocha.js (hunk 1 above)
Comment (skripchenkoVeta) [reply test/0002-CW-do-you-speak-english/Do-you-speak-English.mocha.js:6]: ok, I'll fix it. Thanks

**Review by mTeeeur — APPROVED** [review]
Looks good

**Review by Rascass — APPROVED** [review]
ok

**Review by askomar — APPROVED** [review]
Good

**Pull request merged**